src/formatters/graphml/createGraphML.js

```diff
@@ -55,7 +56,7 @@ const formatAndSerialize = element => formatXml(serialize(element));
 const sha1 = (text) => {
   // eslint-disable-next-line new-cap
   const shaInstance = new jsSHA('SHA-1', 'TEXT', { encoding: 'UTF8' });
-  shaInstance.update(text);
+  shaInstance.update(text.toString());
```

Comment:
This fixes an unreported bug where categorical variables with integer values weren't getting their `key` set correctly.